tests/migrations/test_base.py

```diff
@@ -451,3 +451,27 @@ def assertOptimizesTo(
 
     def assertDoesNotOptimize(self, operations, **kwargs):
         self.assertOptimizesTo(operations, operations, **kwargs)
+
+
+class GenericAliasSerializerTestCase(TestCase):
```

Comment:
Please add your test methods to `WriterTest`. You can see a recent example in #19458.